### PR TITLE
Force error message display

### LINKED ISSUE #77: Failure to load coverage data doesn't show any message
https://github.com/mozilla/code-coverage-reports/blob/f24e1c1e2049af326d1af37e06c8d296a3f61a4b/src/index.js#L120 isn't shown (you can test that by injecting an error in one of the get_ functions).

### PR DESCRIPTION
**Pull request by La0**
Fixes #77

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -117,8 +117,10 @@ async function load() {
       get_history(path),
     ]);
   } catch (err) {
+    console.warn('Failed to load coverage', err);
+    await DOM_READY; // We want to always disaply this message
```
Comment (marco-c) [inline src/index.js]: We could put this in `message` so we don't risk forgetting it in other cases

Comment (La0) [reply src/index.js]: That would make `message` an async function, which is not ideal

Comment (marco-c) [reply src/index.js]: OK, wfm then.

Comment (marco-c) [inline src/index.js]: Typo: display

**Review by marco-c — APPROVED** [review]

**Pull request merged**